docker/debian9/entrypoint.sh

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env bash
```

Comment:
can you reuse this script rather than copy it?

Reply:
> can you reuse this script rather than copy it?

no：
![image](https://user-images.githubusercontent.com/15775037/160320225-f3a3331f-cdd5-45fe-8482-3c7bf61ea4f9.png)